backend/hub/hub.go

```diff
@@ -104,26 +119,51 @@ func (h *hub) HandleAuthentication(
 	conn *websocket.Conn,
 	send chan []byte,
 ) (client identifier.Client, err error) {
+
+	_, message, err := conn.ReadMessage()
```

Comment:
With this new pull request, we now expect a message to come from the frontend regarding tokens.
They MUST say either:

1) ContainsToken: true, in which they'll give a token from the cookie and we can check if it's valid or not
2) ContainsToken: false, in which we'll skip the check and create a new ID from them